tests/codegen/debug-vtable.rs

```diff
@@ -11,8 +11,6 @@
 
 // Make sure that vtables don't have the unnamed_addr attribute when debuginfo is enabled.
 // This helps debuggers more reliably map from dyn pointer to concrete type.
-// CHECK: @vtable.0 = private constant <{
-// CHECK: @vtable.1 = private constant <{
```

Comment:
I don't know whether to be delighted or horrified that this "fix" works to handle both cases here. :)